test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_DeleteConsumerGroupOffsets.cs

```diff
@@ -0,0 +1,120 @@
+            using (var adminClient = new AdminClientBuilder(new AdminClientConfig { BootstrapServers = bootstrapServers }).Build())
+            using (var consumer1 = new ConsumerBuilder<Null, string>(new ConsumerConfig { BootstrapServers = bootstrapServers, GroupId = topic1.Name })
+                .SetPartitionsAssignedHandler((c, partitions) =>
+                {
+                    assignmentDone = true;
+                    Assert.Equal(1, partitions.Count());
+                    Assert.Equal(0, partitions[0].Partition.Value);
+                    Assert.Equal(topic1.Name, partitions[0].Topic);
+                }).Build())
+            {
+                var groupId1 = topic1.Name;
+                var offsetToCommit = 10;
+
+                consumer1.Assign(new List<TopicPartition>() { new TopicPartition(topic1.Name, 0) });
+                consumer1.Commit(new List<TopicPartitionOffset>() { new TopicPartitionOffset(topic1.Name, 0, offsetToCommit) }); //commit some offset for consumer
+
+                var committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(10));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset);
+
+                List<TopicPartition> topicPartitionToReset = new List<TopicPartition>() { new TopicPartition(topic1.Name, 0) };
+                var res = adminClient.DeleteConsumerGroupOffsetsAsync(groupId1, topicPartitionToReset).Result;
+                Assert.Equal(1, res.Count);
+                Assert.Equal(groupId1, res[0].Group);
+                Assert.Equal(1, res[0].Partitions.Count);
+                Assert.Equal(0, res[0].Partitions[0].Partition.Value);
+                Assert.Equal(ErrorCode.NoError, res[0].Partitions[0].Error.Code);
+
+                committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(1));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(Offset.Unset, committedOffsets[0].Offset);
+
+                // Consumer is actively subscribed to the topic
+                assignmentDone = false;
+                consumer1.Subscribe(new List<String>() { topic1.Name });
+                while (!assignmentDone)
+                {
+                    // To handle the rebalance
+                    consumer1.Consume(TimeSpan.FromSeconds(1));
+                }
+                consumer1.Commit(new List<TopicPartitionOffset>() { new TopicPartitionOffset(topic1.Name, 0, offsetToCommit) }); //commit some offset for consumer
+
+                committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(10));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset);
+
+                topicPartitionToReset = new List<TopicPartition>() { new TopicPartition(topic1.Name, 0) };
+                res = adminClient.DeleteConsumerGroupOffsetsAsync(groupId1, topicPartitionToReset).Result;
```

Comment:
i think this should throw an exception.

Reply:
Throwing an exception now, updated the test